esp/src/eclwatch/XrefDirectoriesWidget.js

```diff
@@ -71,14 +71,50 @@ define([
                     store: this.store,
                     columns: {
                         Name: { label: this.i18n.Directory, width: 100, sortable: false },
-                        Num: { label: this.i18n.Files, width: 30, sortable: false },
-                        Size: { label: this.i18n.TotalSize, width: 30, sortable: false },
+                        Num: { label: this.i18n.Files, width: 30, sortable: false,
+                            renderCell: function (object, value, node, options) {
```

Comment:
Rather than `renderCell` (which is a bit heavy), you can specify `className: 'justify-right'`